libs/ledger-live-common/src/hooks/recoverFeatueFlag.ts

```diff
@@ -46,3 +46,12 @@ export function useAlreadySubscribedURI(
 
   return useReplacedURI(uri, id);
 }
+
+export function useUpsellURI(
+  servicesConfig: Feature<any> | null
```

Comment:
`any` is no longer allowed on LLD project, you can possibly use the correct type OR unknown (which will requires you to do a lot of typecheck in the JS if you use unknown)